packages/rule-components/src/RuleTable/RuleTable.js

```diff
@@ -177,9 +179,31 @@ class RuleTable extends Component {
                                 props: { colSpan: columns.length + Boolean(actions) }
                             }]
                         })) :
-                        flatten(createRows(data, columns, expanded, detail)).filter(Boolean)
+                        data.length > 0 ?
+                            flatten(createRows(data, columns, expanded, detail)).filter(Boolean) :
+                            [{
+                                heightAuto: true,
+                                cells: [
+                                    {
+                                        props: { colSpan: 8 },
```

Comment:
This should be length of columns, so anybody can suppply whatever amount of columns they want.